docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -93,6 +93,8 @@ As an owner of a Kafka instance in {product-long-kafka}, you can manage the leve
 
 You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
 
+You also have a set of permissions for all resources within a Kafka instance including topics, groups, transactional IDs, and Access Control Lists (ACLs). This set of permissions cannot be changed and cannot be seen by you or any other user in the *Permission* list in the {product-long-kafka} web console or from Kafka APIs.
```

Comment:
Looks good to me, @AedinC. Suggesting a couple of final, very minor tweaks:

"*As an instance owner*, you *automatically have full access permissions* for all resources within the instance, including...."

Reply:
Thanks John. :)